linera-views/src/common.rs

```diff
@@ -38,6 +38,14 @@ pub(crate) enum Update<T> {
     Set(T),
 }
 
+/// We need to have a maximum key size that handles all possible underlying
+/// sizes. The constraint so far is DynamoDb which has a key length of 1024.
+/// That key length is decreased by 4 due to the use of a value splitting.
+/// Then the KeyValueStoreClient needs to handle some base_key and so we
+/// reduce to 900. Depending on the size, the error can occur in system_api
+/// or in the KeyValueStoreView.
+pub const COMMON_MAX_KEY_SIZE: usize = 900;
```

Comment:
Why "common"? Doesn't this belong to the SDK?

Reply:
If we do that then we establish a dependency of `linera-views` on `linera-sdk`. I do not see that as something we would like to have.
I also think that it may not compile due to circular dependency.

Reply:
Sorry, I understand what you wanted only now.